app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on per-page basis.
+  def full_title page_title = ""
+    base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
`  "Ruby on Rails Tutorial Sample App"` dùng i18n cho text này nhé
tìm hiểu I18n in rails đi

Reply:
done